### ISSUE TITLE
Unable to replay video messages

### ISSUE THREAD

**Issue by rupinderjeet** [GetStream/stream-chat-android#2317]
**Describe the bug**
After a video ends playback, we are unable to play it again without restarting the activity again.

**SDK version**
- 4.17.2

**To Reproduce**
1. Tap a video message.
2. Let the video play out. After this, "play" icon appears.
3. Tap the play icon. Notice that video isn't replayed.
4. Change seek to start of video. ProgressBar appears. But, video is never played again.

**Expected behavior**
After video plays out completely, show a replay icon that replays the video.

**Device:**
 - Vendor and model: Samsung A50 & Emulator Pixel 3
 - Android version: 11 & 10

**Screenshots**

https://user-images.githubusercontent.com/14011726/132453942-3bf5a8e3-9980-4bab-b1cc-fcda1f98574b.mp4

**Extra Information (If any)**
```
E/ExoMediaPlayer: Unable to seek across windows, falling back to in-window seeking
```

**Comment by ogkuzmin** [comment]
Hello, @rupinderjeet 
For playing video we use exoplayer dependency. And I see they have [this issue](https://github.com/brianwernick/ExoMedia/issues/382) in their repository. I see some workarounds in the issue, but it's still not fixed. Not sure that we can deal with it by ourselves. Will wait for the fix from exoplayer maintainers.

**Comment by adasiewiczr** [comment]
Will be fixed in scope of #2880

**Comment by filbabic** [comment]
Closing given that it's resolved in another ticket
